astilectron-bundler/main.go

```diff
@@ -92,6 +93,20 @@ func main() {
 	}
 	c.LDFlags.Merge(astibundler.LDFlags(ldflags))
 
+	if c.VersionElectron == "" {
```

Comment:
Defaults shouldn't be set in the `main` package but in the lib instead. The `New()` function would be a perfect match